src/DotNet/Writer/Metadata.cs

```diff
@@ -2325,7 +2325,7 @@ struct MethodScopeDebugInfo {
 		/// <returns>Its encoded token</returns>
 		protected uint AddResolutionScope(IResolutionScope rs) {
 			if (rs is null) {
-				Error("ResolutionScope is null");
+				// ResolutionScope can now actually be null in .NET 5 Reference Assemblies, so do not throw error any longer here, just return 0
```

Comment:
Since this is allowed according to the spec, this comment can now be removed.

Reply:
Thanks for the feedback, I removed the comment again